Formula/corectl.rb

```diff
@@ -20,19 +20,22 @@ class Corectl < Formula
   def install
     ENV["GOPATH"] = buildpath
 
-    mkdir_p buildpath/"src/github.com/TheNewNormal/"
-    ln_s buildpath, buildpath/"src/github.com/TheNewNormal/#{name}"
+    path = buildpath/"src/github.com/TheNewNormal/#{name}"
+    path.install Dir["*"]
+
     Language::Go.stage_deps resources, buildpath/"src"
 
     args = []
     args << "VERSION=#{version}" if build.stable?
 
-    system "make", "corectl", *args
-    system "make", "documentation/man"
+    cd path do
+      system "make", "corectl", *args
+      system "make", "documentation/man"
```

Comment:
ok. thanks! (sorry if this is obvious, 1 am here) where are you seeing this, the full build log (URL)?. [here](http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/1709/version=yosemite/testReport/brew-test-bot/yosemite/install_corectl/) i see a slightly different thing ...
```
==> Verifying corectl-
...[truncated 99411 chars]...
parent directories): .git
./corectl utils mkMan
inappropriate ioctl for device
ls: documentation/man/*.1: No such file or directory
Warning: tried to install empty array to /usr/local/Cellar/corectl/0.5.5/share/man/man1
==> Cleaning
```
being that i have no idea where the `inappropriate ioctl for device` comes from. (`brew install  Formula/corectl.rb --no-clean -v ` works flawlessly locally. 

Thanks in advance for your patience!